extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -236,11 +242,7 @@ function _checkIfCanAddMeasurementsToDisplaySet(
     measurement => measurement.loaded === false
   );
 
-  if (
-    unloadedMeasurements.length === 0 ||
-    !(newDisplaySet instanceof ImageSet) ||
-    newDisplaySet.unsupported
-  ) {
+  if (unloadedMeasurements.length === 0 || newDisplaySet.unsupported) {
```

Comment:
I think you need to check if its image otherwise its not guaranteed that the ds wont have have .images